cli/app.go

```diff
@@ -442,6 +442,155 @@ var app = &cli.App{
 						},
 					},
 				},
+				{
```

Comment:
Overall, I think it's kind of odd to have this as a subcommand of data. The ML training service operates largely as separate from the data service and the only API in ML training that relates to data directly is training. The get/list/cancel APIs actually don't interact with data at all. What do you think about separating out the train APIs to their own command, maybe `model` or `train`?

Reply:
I think the delineation between this and data will become even starker once we start accepting only dataset IDs for training.

Reply:
Retested all commands and updated in the PR description.